src/main/scala/com/github/sstone/amqp/Amqp.scala

```diff
@@ -75,7 +75,7 @@ object Amqp {
    */
   case class ChannelParameters(qos: Int, global: Boolean = false)
 
-  case class Binding(exchange: ExchangeParameters, queue: QueueParameters, routingKey: String)
+  case class Binding(exchange: ExchangeParameters, queue: QueueParameters, routingKeys: Set[String])

```

Comment:
I think it's preferable to use varargs instead of a `Set` here, as it will make the declaration of `Binding`s simpler. The same applies to `QueueBind` below.

Reply:
The approach discussed offline (re-ordering arguments) won't work because of the default argument value in:
`case class QueueBind(queue: String, exchange: String, routingKeys: Set[String], args: Map[String, AnyRef] = Map.empty) extends Request`

This:
`case class QueueBind(queue: String, exchange: String, args: Map[String, AnyRef] = Map.empty, routingKeys: String*) extends Request`

is not allowed.

I'm thinking it might be best to leave this as is. Any ideas?

Reply:
Let's leave it as is then.